system/include/emscripten/wasmfs.h

```diff
@@ -25,7 +25,10 @@ backend_t wasmfs_get_backend_by_fd(int fd);
 // Creates a JSFile Backend in the new file system.
 backend_t wasmfs_create_js_file_backend();
 
-// Creates a file in a specific backend and returns an fd to an open file.
+// Creates a Proxied Backend in the new file system.
+backend_t wasmfs_create_proxied_file_backend(backend_t backend);
```

Comment:
The comment says "proxied backend" while the function says "proxied file backend". If we only proxy file operations then the latter sounds better. But I think we must proxy directory ops too - they can also only be handled on the dedicated thread when using access handles etc. So how about `wasmfs_create_proxied_backend`?

Reply:
Good point about supporting directories as well. Done.